doc/classes/Basis.xml

```diff
@@ -4,10 +4,12 @@
 		A 3×3 matrix for representing 3D rotation and scale.
 	</brief_description>
 	<description>
-		A 3×3 matrix used for representing 3D rotation and scale. Usually used as an orthogonal basis for a [Transform3D].
-		Contains 3 vector fields X, Y and Z as its columns, which are typically interpreted as the local basis vectors of a transformation. For such use, it is composed of a scaling and a rotation matrix, in that order (M = R.S).
-		Basis can also be accessed as an array of 3D vectors. These vectors are usually orthogonal to each other, but are not necessarily normalized (due to scaling).
+		The [Basis] built-in [Variant] type represents a 3x3 [url=https://en.wikipedia.org/wiki/Matrix_(mathematics)]matrix[/url] used for 3D rotation, scale, and shearing. It is frequently used within a [Transform3D].
+		A [Basis] is composed by 3 axis vectors, each representing a column of the matrix: [member x], [member y], and [member z]. The length of each axis ([method Vector3.length]) influences the basis's scale, while the direction of all axes influence the rotation. Usually, these axes are perpendicular to one another. However, when you rotate any axis individually, the basis becomes sheared. Applying a sheared basis to a 3D model will make the model appear distorted.
+		A [Basis] is [b]orthogonal[/b] if its axes are perpendicular from each other, which allows it to represent rotations. A basis is [b]normalized[/b] if the length of every axis is [code]1[/code]. A basis is [b]uniform[/b] if all axes share the same length (see [method get_scale]).
 		For a general introduction, see the [url=$DOCS_URL/tutorials/math/matrices_and_transforms.html]Matrices and transforms[/url] tutorial.
+		[b]Note:[/b] Godot uses the [b]right-hand coordinate system[/b] (+X is right, +Y is up, and +Z is back). This may be different from the coordinate system you are used to. For example, by convention, 3D assets are rotated towards the camera (+X is [b]left[/b], +Y is up, and +Z is [b]front[/b]). For more information, see the [url=$DOCS_URL/tutorials/assets_pipeline/importing_scenes.html#d-asset-direction-conventions]Importing 3D Scenes[/url] tutorial.
```

Comment:
This is misleading, coordinate system handedness has nothing to do with the conventions of how assets are oriented. They are separate things.

To make an analogy, it's like saying "The USA uses US dollars. This may be different from what you are used to, for example, they use feet and inches instead of meters". That statement would imply that US dollars imply feet and inches. No, they are completely separate things.

Reply:
What could be said in this case, then?

Reply:
Actually, looking at this closer, Basis actually has no inherent handedness. I'm pretty sure that all of the same math methods will work exactly as-is if you used the same Basis code with a left-handed coordinate system. What makes Godot right-handed is its interpretation of the data we store in Basis.

The only places where conventions are encoded into Basis are 1) In the default Euler angle order YXZ which works best for Y-vertical, X-sides, Z-forward/back systems, and 2) The directions used by the `looking_at` method.

Reply:
So the note about the coordinate system would be best suited for Transform3D, instead? Omitting it here may conflict with what @TokageItLab requested to mention above.

Reply:
> Actually, that definition may not be necessary until rendering, but if you are trying to draw/link a figure, you will need to annotate that the coordinate system of the Godot and the figure are same.

As I said above, no coordinate system definition is required until rendering. However, since you seem to have links to some figures, so it would be better to indicate whether the figure and the result rendered by Godot match or not.

Reply:
To clarify, the preamble I wrote in the asset direction conventions section that Tokage linked is prerequisite information to explaining the asset direction conventions in terms of that coordinate system. So it's "Godot is X, and in terms of X, Godot is Y" not "Godot is X, and because of X, Godot is Y". An immediate minor improvement would be to remove the "For example" prefix to that sentence.

In terms of the documentation of the Basis structure, we need to make it clear specifically what documentation applies to Basis itself, what applies to its interpretation, and what applies to a specific method.

It may also be useful to mention specifically what engines don't use right-handed, instead of a vague notion of not being what you're used to. Right-handed is the dominant system, it's just a few outliers like Unity and Unreal that use left-handed.

I recommend that the text be changed to this:

```
Godot uses a [url=https://en.wikipedia.org/wiki/Right-hand_rule]right-handed coordinate system[/url]. Most software is right-handed, but this is different from the Unity and Unreal engines. Built-in types like [Camera3D] use the convention of +X is right, +Y is up, and +Z is back. Other objects within this coordinate system may use different direction conventions. For more information, see the [url=$DOCS_URL/tutorials/assets_pipeline/importing_scenes.html#d-asset-direction-conventions]Importing 3D Scenes[/url] tutorial.
```

Feel free to tweak if needed. Any further elaboration of direction conventions should be localized to the description of the `looking_at` method IMO.